packages/components/src/components/primitives/Slider/types.ts

```diff
@@ -1,13 +1,17 @@
+import { FC } from 'react';
 import {
   SliderProps as ChakraSliderProps,
-  SliderTrackProps as ChakraSliderTrackProps,
-  SliderThumbProps as ChakraSliderTumbProps,
-  BoxProps,
+  SliderTrackProps,
+  SliderInnerTrackProps,
+  SliderThumbProps,
+  SliderMarkProps,
 } from '@chakra-ui/react';
 
-// SliderFilledTrack props matches with Box props
-export interface SliderProps extends ChakraSliderProps {
-  filledTrackProps?: BoxProps;
-  trackProps?: ChakraSliderTrackProps;
-  thumbProps?: ChakraSliderTumbProps;
+export type SliderProps = Omit<ChakraSliderProps, 'size'>;
```

Comment:
I omit size props since the design doesn't it, according the our rules we can't do this:
`export interface SliderProps extends Omit<ChakraSliderProps, 'size'>{};` and besides lint doesn't like that. So I decided to use `type` instead of `interface` to avoid lint errors.

Reply:
This is the right approach